src/EditorFeatures/CSharpTest/CodeActions/ExtractMethod/ExtractMethodTests.cs

```diff
@@ -1423,7 +1423,7 @@ private static string NewMethod(int i)
 
         return v;
     }
-}", options: Option(CSharpCodeStyleOptions.UseImplicitTypeForIntrinsicTypes, CodeStyleOptions.TrueWithSuggestionEnforcement));
+}", options: Option(CSharpCodeStyleOptions.VarForBuiltInTypes, CodeStyleOptions.TrueWithSuggestionEnforcement));
```

Comment:
these are the names we use for our actual editorconfig settings.  i'm trying to unify them all so we can have a consistent way of talking about the options from the UI, to editorconfig, to our internal enums/options.